Makefile

```diff
@@ -16,7 +16,7 @@ $(GIT_HOOKS):
 include common.mk
 
 CFLAGS = -I./include
-CFLAGS += -std=gnu99 -Wall -W
+CFLAGS += -std=c11 -Wall -W
```

Comment:
I would make use of GNU extensions for this project. So, please revise as `gnu11`.